pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -84,15 +84,20 @@
     <RuntimeGroup Include="ol">
       <Parent>rhel</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6</Versions>
+      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6;7.7;7.8;7.9;7.10</Versions>
       <ApplyVersionsToParent>true</ApplyVersionsToParent>
     </RuntimeGroup>
     <RuntimeGroup Include="ol">
       <Parent>rhel</Parent>
-      <Architectures>x64</Architectures>
-      <Versions>8;8.0</Versions>
+      <Architectures>x64;arm64</Architectures>
+      <Versions>8;8.0;8.1;8.2;8.3;8.4;8.5;8.6</Versions>
       <ApplyVersionsToParent>true</ApplyVersionsToParent>
     </RuntimeGroup>
+    <RuntimeGroup Include="ol">
+      <Parent>rhel</Parent>
+      <Architectures>x64;arm64</Architectures>
+      <Versions>9;9.0;9.1</Versions>
```

Comment:
Are you actually planning to build .NET Core on Oracle Linux 9? If so, do you have a plan to deal with OpenSSL 3.0 being the only version available to build against?